README.md

```diff
@@ -238,6 +240,18 @@ assert({}); // fail
 assert(() => {}); // pass
 ```
 
+### v.date
+
+Passes if input is a `Date` that is valid (`input.toString() !== 'Invalid Date'`).
```

Comment:
It might not be clear to everyone what all inputs correspond to `'Invalid Date'`. 
Do you have think we should add a link to some documentation / blog which digs deeper into this topic?

Reply:
I have poked around and can't find a good reference for this. If anyone finds one we can add later, I guess.